### PR TITLE
[3006.x] Return Not Found when service is missing

### PR DESCRIPTION
**Pull request by twangboy**
### What does this PR do?
Fixes issue with the win_system module that returned a CommandExecutionError if the service was not found on the system. It now returns "Not Found".

Additionally, ports test_win_service to pytests.

### What issues does this PR fix or reference?
Fixes: #63577 

### Merge requirements satisfied?
**[NOTICE] Bug fixes or features added to Salt require tests.**

- [x] Docs
- [x] Changelog - https://docs.saltproject.io/en/master/topics/development/changelog.html
- [x] Tests written/updated

### Commits signed with GPG?
Yes

### CODE REVIEW

File: salt/modules/win_service.py [hunk 1]
```diff
@@ -486,12 +486,16 @@ def status(name, *args, **kwargs):
     .. versionchanged:: 2018.3.0
         The service name can now be a glob (e.g. ``salt*``)
 
+    .. versionchanged:: 3006
```
Comment (s0undt3ch) [inline salt/modules/win_service.py:489]:
```suggestion
    .. versionchanged:: 3006.0
```

**Review by s0undt3ch — CHANGES_REQUESTED** [review]

File: salt/modules/win_service.py (hunk 1 above)
Comment (twangboy) [reply salt/modules/win_service.py]: Fixed

File: changelog/63577.fixed.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Issue #63577: ``service.status`` on Windows does no longer throws a
```
Comment (s0undt3ch) [inline changelog/63577.fixed.md:1]:
```suggestion
``service.status`` on Windows does no longer throws a
```

**Review by s0undt3ch — CHANGES_REQUESTED** [review]
Sorry, should have also see this on the first review.
The issue will get linked, no need to prefix the changelog entry with it.

File: changelog/63577.fixed.md (hunk 2 above)
Comment (twangboy) [reply changelog/63577.fixed.md]: Done

**Review by s0undt3ch — APPROVED** [review]

**Pull request merged**